services/api/src/resources/project/resolvers.js

```diff
@@ -215,6 +215,25 @@ const addProject = async (
   await KeycloakOperations.addGroup(project);
   await SearchguardOperations.addProject(project);
 
+  await Helpers.mapIfNoDirectProjectAccess(
+    R.prop('id', project),
+    cid,
+    async ({
+      keycloakUserId,
+      keycloakUsername,
+      keycloakGroupId,
+      keycloakGroupName,
+    }) => {
+      await keycloakClient.users.addToGroup({
```

Comment:
can we use `KeycloakOperations.addUserToGroup` (or maybe create another function as this one does quite some searching for userid and groupid)? I would like to not use `keycloakClient` in the `resolvers.js` anymore and abstract all Keycloak operations through `KeycloakOperations`

Reply:
I don't see any way to avoid querying a lot for keycloak user/project information, since the keycloak-admin library doesn't return the data that was created (even the [tests](https://github.com/Canner/keycloak-admin/blob/master/test/groups.spec.ts#L26) run a `create` then a `find`). I did abstract away the map function in a Helper.